recipes/libgd/all/conandata.yml

```diff
@@ -0,0 +1,18 @@
+sources:
+  "2.2.5":
+    url: "https://github.com/libgd/libgd/releases/download/gd-2.2.5/libgd-2.2.5.tar.gz"
+    sha256: "a66111c9b4a04e818e9e2a37d7ae8d4aae0939a100a36b0ffb52c706a09074b5"
+  "2.3.0":
+    url: "https://github.com/libgd/libgd/archive/gd-2.3.0.tar.gz"
+    sha256: "a77dfbbf8bfa7f19c935c11f3b939321f8c1059953a91203158cb2dbf27a0f9e"
+patches:
+  "2.2.5":
+    - patch_file: "patches/2.2.5-msvc-static-lib.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/2.2.5-qualify-nondll.patch"
+      base_path: "source_subfolder"
```

Comment:
these 2 patches are redundant

Reply:
I did it this way to factor the patches as to what they do: the `2.2.5-qualify-nondll.patch` and `qualify-nondll.patch` are essentially the same but the surrounding context was different enough that I couldn't reuse the patch for both versions.

2.3.0 also has two patches.

I think this is a difference of opinion: Mine is that smaller patches with descriptive names are easier to understand, for the same reason that this PR has more than one commit.

Reply:
I believe it's harder to understand, because these two patches in 2.2.5 are dealing with the same issue.

Your fist patch remove one line and introduce a bunch of code. The second patch replaces some parts you have introduced in the first patch.